iceoryx_posh/test/integrationtests/test_mq_interface_startup_race.cpp

```diff
@@ -106,13 +106,13 @@ class CMqInterfaceStartupRace_test : public Test
     /// @note smart_lock in combination with optional is currently not really usable
     std::mutex m_roudiQueueMutex;
     IpcChannelType::result_t m_roudiQueue{
-        IpcChannelType::create(roudi::MQ_ROUDI_NAME, IpcChannelMode::BLOCKING, IpcChannelSide::SERVER)};
+        IpcChannelType::create(roudi::IPC_CHANNEL_ROUDI_NAME, IpcChannelMode::BLOCKING, IpcChannelSide::SERVER)};
     std::mutex m_appQueueMutex;
     IpcChannelType::result_t m_appQueue;
 };
 
 #if !defined(__APPLE__)
-TEST_F(CMqInterfaceStartupRace_test, ObsoleteRouDiMq)
+TEST_F(CMqInterfaceStartupRace_test, DISABLED_ObsoleteRouDiMq)
```

Comment:
we have to check which of these make sense after the startup procedures is redone